hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1875,16 +1875,17 @@
     // ENTROPY_SRC (pre-verified IP) integration tests:
     {
       name: chip_sw_entropy_src_ast_rng_req
+      //TODO: A dv enginner should add an asertion to check the connectivity for this test.
```

Comment:
Could you file an issue indicating the desired connection?

Reply:
done